codebreaker_garage/spec/codebreaker_garage/console_spec.rb

```diff
@@ -2,33 +2,36 @@
 
 module Codebreaker_garage
   RSpec.describe Console do
-    context '#start' do
+
+    describe "#start" do
       before do
-        allow(subject.instance_variable_get(:@game)).to receive(:any_attempts?).and_return(false)
+        subject.instance_variable_set(:@user_tries_to_guess, '1234')
+        allow(subject.instance_variable_get(:@game)).to receive(:has_attempts?).and_return(true)
+      end
+
+      it "says that game has been started" do
+        allow(subject).to receive(:gets).and_return('exit')
+        expect { subject.start }.to output(/New game has been started/).to_stdout
+      end
+
+      it "print mark answer" do
+        allow(subject).to receive(:gets).and_return('1234')
+        allow(subject.instance_variable_get(:@game)).to receive(:correct_input).and_return('----')
+        expect { subject.start }.to output(/----/).to_stdout
```

Comment:
you have loop in this method, it quite possible that it is endless. could you check please?